core/src/test/java/hudson/util/CopyOnWriteListTest.java

```diff
@@ -50,20 +53,24 @@ public void serialization() {
         TestData td = new TestData();
 
         String out = xs.toXML(td);
-        assertEquals("empty lists", "<hudson.util.CopyOnWriteListTest_-TestData>"
-                + "<list1/><list2/></hudson.util.CopyOnWriteListTest_-TestData>",
-                out.replaceAll("\\s+", ""));
+        String expected = "<hudson.util.CopyOnWriteListTest_-TestData>"
+                + "<list1/><list2/></hudson.util.CopyOnWriteListTest_-TestData>";
+        assertThat(out, CompareMatcher.isSimilarTo(expected).ignoreWhitespace()
```

Comment:
normally you would static import a matcher, as it's supposed to be read as a sentence

> assert that out is similar to expected

```suggestion
        assertThat(out, isSimilarTo(expected).ignoreWhitespace()
```